tasks/main.yml

```diff
@@ -19,3 +19,18 @@
 
 - include: awx-install-playbook.yml
   when: awx_run_install_playbook
+
+- name: Add vagrant user to docker group
```

Comment:
This would be highly specific to using this role in a Vagrant environment. If this is for the ansible-vagrant-examples repository, it should be made to happen there. Also note that the Docker role itself handles adding users to the Docker group—see the https://github.com/geerlingguy/ansible-role-docker docs, specifically the `docker_users` variable.

Reply:
That makes perfect sense, forgot this is abstracted away from Vagrant. Updated to remove this specific step. 

To my knowledge the other additions are still useful - AWX is somewhat buggy and requires a container service (I believe just the awx_task container) after about 30 seconds for it to come up properly.